### DISCUSSION TITLE
Adding fan speed sensor to tray icons?

### DISCUSSION THREAD

**Discussion by x1l0n** [Rem0o/FanControl.Releases#2540] [Q&A] [answered]
Is it possible to add a fan speed sensor (%) to the tray icons? Currently in V191 I can only see the temperature sensors listed in the "tray icons" section. Maybe there is a workaround way to add it?

**Comment by Rem0o** [comment]
The tray icon width is limited to 2 char for readable fonts.  RPM wouldn't fit.

**Reply by x1l0n** [reply]
Sorry for not clarifying earlier - I meant adding the percentage value, not the RPM. So 0-99 would suffice.

**Reply by Rem0o** [reply]
What about 100%?

**Reply by x1l0n** [reply, marked as answer]
Even 00 for 100 (and a single 0 for 0%, respectively) would be better than no Icon ;) Currently I use msi afterburner+rivatuner stats server to achieve the same for tray icons, but I would much rather prefer to only use FanControl.

**Reply by Rem0o** [reply]
Sold.
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/68af7aaf-a6cb-4f14-8df9-783728ecd450)

**Reply by x1l0n** [reply]
Thank you so much for the AWESOME support! V192 not only added support for % fan speeds in the tray icons, but also support for 3digits in the tray icon!
Now I can easily take just a glance at the tray bar and know the gpu fan speeds, gpu temp, cpu temp, main fan controller speeds (and much more if needed) just with FanControl! l
![image](https://github.com/Rem0o/FanControl.Releases/assets/36621241/9d21088c-cc98-4968-a4de-4a75c17d4858)
